src/status_im2/contexts/chat/lightbox/view.cljs

```diff
@@ -87,6 +86,14 @@
        #(toggle-opacity opacity-value border-value transparent? index atoms)]
       [rn/view {:style {:width seperator-width}}]])])
 
+;; using `safe-area/consumer` in this component in iOS causes unnecessary re-renders and weird behaviour
```

Comment:
Thanks for adding a comment. It would be good to fix this, since re-renders are often caused by issues on our side (not on Reagent, React, etc).